zio-cli/jvm/src/test/scala/zio/cli/FigFontParserSpec.scala

```diff
@@ -0,0 +1,27 @@
+package zio.cli
+
+import zio._
+import zio.blocking.effectBlockingIO
+import zio.test.Assertion._
+import zio.test._
+
+object FigFontParserSpec extends DefaultRunnableSpec {
+  override def aspects = List(TestAspect.jvmOnly)
+
+  override def spec = suite("parsing")(
+    testM("parse standard.flf") {
```

Comment:
This is good. Maybe it would be nice to add a test that is normally `@@ ignored`, and which downloads all fonts from the website and tests that all of them can be parsed?

Reply:
Thinking of a resource like this:

```
http://www.figlet.org/fontdb.cgi
```

Wouldn't be too hard to just `ZIO.foreach` on a list of these files, `scala.io.Source` load them, and then parse them. Could catch possible errors and be run manually from time to time if anyone works in the parser code.

Reply:
Right, i had something like this in mind, and few related questions for zio-test: 1) to check `@@ ignored` 2) in my practice i found useful for such tasks to generate a report for visual inspection, hence how to better do it in zio-test, e.g. to gen a temp file, and log a URL in console -- does zio-test have a test-scoped log channel for that?

Reply:
ZIO Test doesn't have report generation yet, but will soon.